### ISSUE TITLE
Hyperparameters for other models than EDM.

### ISSUE THREAD

**Issue by mk322** [mingyuanzhou/SiD#5]
Hi,

Really nice work! I am working on adapting SiD into other diffusion models (e.g., CogVideoX). I saw that there are several hyperparameters in noise scheduler of SiD. I am wondering if these are exclusive for EDM, such as σmin = 0.002, σmax = 80, ρ = 7.0, σinit=0.25. If I use these number, I cannot use the original noise DDIM scheduler of CogVideoX. Also, when we are trying to using other models, are there any other hyparameters we need to care about? I would really appreciate your help.

**Comment by mingyuanzhou** [comment]
Thanks for your interest! 

There are two commonly used noise schedules: the EDM style and the DDPM/DDIM style.

For models pretrained using the EDM style, the SiD and SiDA code bases should work with minimal modifications.

For models pretrained using the DDPM style, please refer to our guided SiD code base (SiD-LSG): [[SiD-LSG GitHub Repository](https://github.com/mingyuanzhou/SiD-LSG)](https://github.com/mingyuanzhou/SiD-LSG).

If your code is free of bugs, the default hyperparameters in SiD, SiDA, and SiD-LSG are likely to work out of the box on another pretrained diffusion model. However, you may still need to adjust standard optimizer hyperparameters, such as the learning rates for G and the fake score network, loss scaling (especially for FP16, though less critical for FP32 and AMP (the AMP update will be included in the upcoming SiD-LSG update)), and minibatch size.
